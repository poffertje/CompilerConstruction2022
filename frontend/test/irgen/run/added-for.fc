extern int printf(char[] format, ...);

int main() {
    for (int i = 0 to 100){
        if (i % 3 == 0) {
            i += 10;
        }

        if (i == 20) {
            i = i - 1;
            i = i * 2;
        }
            
        printf("%d \n", i);
    }

    return 0;
}