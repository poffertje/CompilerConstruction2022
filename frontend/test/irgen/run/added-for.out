10 
11 
22 
23 
34 
35 
46 
47 
58 
59 
70 
71 
82 
83 
94 
95 
106 
